build/package/chocolatey/tools/LICENSE.txt

```diff
@@ -0,0 +1,11 @@
+﻿
+Note: Include this file if including binaries you have the right to distribute. 
+Otherwise delete. this file.
+
+===DELETE ABOVE THIS LINE AND THIS LINE===

```

Comment:
Can we edit the nuspec `<files>` section to pull this from the root and delete this file?

Reply:
I just copied the file in during build.